README.md

```diff
@@ -118,8 +130,254 @@ Success-Response:
     "email": "amanda@gmail.com",
     "phone": "24976520981",
     "address": "46170-000",
-    "city": "Rio de Constas",
+    "city": "Rio de Contas",
     "state": "Bahia"
   }
 ]
 ```
+---
+## <center>**Forecast Risk** <a name="forecast-risk"></a></center>
+---
+Send a list of cities, states and precipitation and returns a list of endangered cities along with a quantity of users in the cities. The precipitation values and which values correspond to a danger were defined in the code.
+
+- Not authenticated (?)
+## Parameter
+
+| **Field**         | **Type**          | **Description**                      |
+| ------------------| ------------------| -------------------------------------|
+| city              | string            | A city name                          |
+| state             | string            | The city state                       |
+| precipitation     | integer or float  | A precipitation integer value in mm  |
+
+**Body:**
+
+```json
+[
+  {
+		"city": "Petrópolis",
+		"state": "Rio de Janeiro",
+		"precipitation": 80
+	},
+	{
+		"city": "São Paulo",
+		"state": "São Paulo",
+		"precipitation": 30
+	}
+]
+```
+
+Success-Response:
+
+**HTTP/1.1**: 200 **OK**
+
+
+```json
+{
+  "endangered_cities": [
+    {
+      "city": "Petrópolis",
+      "state": "Rio de Janeiro",
+      "users_warned": 2
+    }
+  ]
+}
+```
+
+### Errors
+
+<details>
+<summary>Forecast risk request with missing keys</summary> <a name="forecast-risk-missing-keys"></a>
+
+**Body:**
+
+```json
```

Comment:
I've added a body in the errors, but if someone think this is not necessary i can delete it.